packages/ckeditor5-fullscreen/tests/handlers/abstracteditor.js

```diff
@@ -3,32 +3,62 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
  */
 
+/* globals document */
+
 import global from '@ckeditor/ckeditor5-utils/src/dom/global.js';
 import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
 import { Paragraph } from '@ckeditor/ckeditor5-paragraph';
 import { Essentials } from '@ckeditor/ckeditor5-essentials';
+import { PresenceListUI, PresenceList } from '@ckeditor/ckeditor5-real-time-collaboration';
+import { DocumentOutline, DocumentOutlineUI } from '@ckeditor/ckeditor5-document-outline';
+import mockCloudServices, { CloudServicesMock } from '@ckeditor/ckeditor5-real-time-collaboration/tests/_utils/mockcloudservices.js';
 
 import AbstractEditorHandler from '../../src/handlers/abstracteditor.js';
 
 describe( 'AbstractHandler', () => {
-	let abstractHandler, domElement, editor;
+	let abstractHandler, domElement, editor, presenceList, documentOutline;
+
+	mockCloudServices();
 
 	beforeEach( async () => {
 		domElement = global.document.createElement( 'div' );
 		global.document.body.appendChild( domElement );
+		presenceList = document.createElement( 'div' );
+		document.body.appendChild( presenceList );
+		documentOutline = document.createElement( 'div' );
+		document.body.appendChild( documentOutline );
 
 		editor = await ClassicEditor.create( domElement, {
 			plugins: [
 				Paragraph,
-				Essentials
-			]
+				Essentials,
+				PresenceListUI,
+				PresenceList,
+				CloudServicesMock,
+				DocumentOutline,
+				DocumentOutlineUI
+			],
+			cloudServices: {
+				tokenUrl: 'abc',
+				webSocketUrl: 'web-socket-url'
+			},
+			collaboration: {
+				channelId: 'test'
+			},
+			presenceList: {
+				container: presenceList
+			},
+			documentOutline: {
+				container: documentOutline
+			}
```

Comment:
For simplicity, I'd keep the config here minimalistic and create more complex editors on demand in tests that need them. But here as I wrote before we don't want to test the commercial features anyway.